app/AboutViewController.m

```diff
@@ -58,7 +58,7 @@ - (void)viewDidLoad {
             [self _updateUI];
         });
     }];
-    [NSNotificationCenter.defaultCenter addObserver:self selector:@selector(_updateUI) name:FsUpdatedNotification object:nil];
+    [NSNotificationCenter.defaultCenter addObserver:self selector:@selector(_updateUI:) name:FsUpdatedNotification object:nil];
```

Comment:
How important is this?

Reply:
In theory the selector takes a parameter that you are just ignoring completely from the signature, which is bad. In practice it's been working, of course.